### PR TITLE
DOCSP-45921-export-query-prep

### PR DESCRIPTION
**Pull request by ajhuh-mdb**
## DESCRIPTION
- Swapped out procedure to use the `.. procedure::` directive
  - Updated procedures to reflect current UI experience (was outdated)
  - New images for updated UI

Other bug updates: 
- Moved open-new-tab steps yaml file to shared includes directory.
- Added Compass-only tags to Data Lake note on Query Plan page + compass isolated version note on Schema page

## STAGING
- [Export Query to Language](https://deploy-preview-703--docs-compass.netlify.app/export-query-to-language/)
- [Tabbed View](https://deploy-preview-703--docs-compass.netlify.app/collections/#tabbed-view)

## JIRA
https://jira.mongodb.org/browse/DOCSP-45921

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [ ] Is this free of spelling errors?
- [ ] Is this free of grammatical errors?
- [ ] Is this free of staging / rendering issues?
- [ ] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

File: source/includes/shared/steps-open-new-tab.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+title: Click the caret icon next to the database which contains the
```
Comment (mdb-ashley) [inline source/includes/shared/steps-open-new-tab.yaml:1]:
```suggestion
title: Click the caret icon next to the database that contains the
```

File: source/includes/shared/steps-open-new-tab.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  collection you wish to view.
```
Comment (mdb-ashley) [inline source/includes/shared/steps-open-new-tab.yaml:2]:
```suggestion
  collection you want to view.
```

**Review by mdb-ashley — APPROVED** [review]
LGTM! Just a couple comments for your consideration. Thanks, Ali.

File: source/includes/shared/steps-open-new-tab.yaml (hunk 1 above)
Comment (ajhuh-mdb) [reply source/includes/shared/steps-open-new-tab.yaml]: updated!

File: source/includes/shared/steps-open-new-tab.yaml (hunk 2 above)
Comment (ajhuh-mdb) [reply source/includes/shared/steps-open-new-tab.yaml]: updated!

**Pull request merged**